mobify/progressive-web-scaffold
# Astro: Merlins checkout app event showing sign in and showing shop

Astro: Merlins checkout app event showing sign in and showing shop
from #300 create app event that shows the correct tab.

 **JIRA**: https://mobify.atlassian.net/browse/HYB-985
 **Linked PRs**: #300

## Changes
- create app event called `showSignIn` and `showShop` that causes tabBarController to select account or shop tab

## How to test-drive this PR
- open up checkout cart modal
- click signIn, the modal should close and open signIn tab
- click continue Shopping, the modal should close and open the shop tab
